sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/Azure.ResourceManager.MachineLearningServices.csproj

```diff
@@ -7,17 +7,6 @@
     </Description>
     <PackageId>Azure.ResourceManager.MachineLearningServices</PackageId>
     <PackageTags>azure;management;machinelearning;machinelearningservices;aml;ml</PackageTags>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);AZC0001;AZC0100;CS1591;IDT003;</NoWarn>
```

Comment:
IDT003 was about invalid `<inheritdoc />` tags being added to non-override methods. I got a few build errors [for example](https://dev.azure.com/azure-sdk/public/_build/results?buildId=923646&view=logs&j=7e6a3fb7-dbfe-5169-4db8-92b72295ba6c&t=f10a3734-99d3-576b-5dfc-2239b92c5880)
Should be able to remove after (PR TBD) is merged.